packages/manager/modules/nasha/src/order/order.controller.js

```diff
@@ -0,0 +1,134 @@
+import JSURL from 'jsurl';
+import { uniq, uniqBy } from 'lodash';
+import { CatalogPricing } from '@ovh-ux/manager-models';
+
+export default class NashaOrderController {
+  /* @ngInject */
+  constructor($window, coreConfig, BillingService, RedirectionService) {
+    this.$window = $window;
+    this.BillingService = BillingService;
+
+    this.user = coreConfig.getUser();
+    this.userLocale = coreConfig.getUserLocale();
+    this.expressOrderUrl = RedirectionService.getURL('expressOrder');
+
+    this.diskType = {
+      value: undefined,
+      focused: false,
+      list: [],
+    };
+    this.capacity = {
+      value: undefined,
+      focused: false,
+      plans: [],
+    };
+    this.datacenter = {
+      value: undefined,
+      focused: false,
+      list: [],
+    };
+    this.commitment = {
+      value: undefined,
+      focused: false,
+      pricings: [],
+      price: null,
+    };
+    this.payment = {
+      value: undefined,
+      focused: false,
+      modes: [],
+    };
+  }
+
+  get plan() {
+    return this.capacity.value;
+  }
+
+  set plan(plan) {
+    this.capacity.value = plan;
+  }
+
+  onDiskTypeFocus() {
+    this.focus('diskType');
+    this.diskType.list = uniq(this.plans.map(({ diskType }) => diskType));
+  }
+
+  onCapacityFocus() {
+    this.focus('capacity');
+
+    this.capacity.plans = uniqBy(
+      this.plans.filter(({ diskType }) => diskType === this.diskType.value),
+      'capacity.value',
+    );
+
+    this.capacity.plans.sort(
+      ({ capacity: { value: a } }, { capacity: { value: b } }) => a - b,
```

Comment:
**praise:** Very good use of destructuring.

Reply:
not mandatory but since you are alreading using lodash you can simply use sortBy(this.capacity.plans, 'capacity.value')